backend/eslint.config.js

```diff
@@ -0,0 +1,61 @@
+//
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+const neostandard = require('neostandard')
+const pluginJest = require('eslint-plugin-jest')
+const pluginSecurity = require('eslint-plugin-security')
+const pluginLodash = require('eslint-plugin-lodash')
+const pluginImport = require('eslint-plugin-import')
+
+module.exports = [
+  ...neostandard({}),
+  {
+    rules: {
+      '@stylistic/comma-dangle': ['error', 'always-multiline'],
+      'no-console': 'error',
+      'no-debugger': 'error',
+    },
+  },
+  pluginSecurity.configs.recommended,
+  {
+    plugins: {
+      import: pluginImport,
+    },
+    rules: pluginImport.flatConfigs.recommended.rules,
+  },
```

Comment:
just `pluginImport.flatConfigs.recommended`?

Reply:
The config sets `languageOptions.ecmaVersion` to `2018`. But we need `2022`